File: PYTHON/Numeric-Functions/factorial.py
def factorial (num):
	num = int(num)
	res = int(1)
	while (num > 0):
		res = int(num * res)
		num = int(num - 1)
	return res

x = int(input("User, enter a number: "))
results = factorial(x)
print(str(x) + " --> " + str(results))

File: C#/code/factorial.cs
namespace aviv_yunker {
	class factorial {
		public static void Main (string [] args) {
			string x;
			int num;
			int res = 1;
			System.Console.WriteLine("User, enter a number: ");
			x = System.Console.ReadLine();
			num = int.Parse(x);
			while (num > 0) {
				res *= num;
				num --;
			}
			System.Console.WriteLine("{0} --> {1}", x, res);
		}
	}
}
